### DISCUSSION TITLE
add `not` operator to filters

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#9857] [Feature Requests]
```json
{
	"_and": [
		{
			"owner": {
				"_eq": "$CURRENT_USER"
			}
		},
		{
			"status": {
				"_in": ["published", "draft"]
			}
		}
	]
}
```

It'd be cool to be able to negate _logical operators_:

```json
"_and:not": [
```

---

It could also be applied to _filter operators_:

```json
"_eq:not": "$CURRENT_USER"
```

**Comment by rijkvanzanten** [comment]
Most filter operators already have a negative equivalent: `eq` / `neq`, `between` / `nbetween` etc (https://docs.directus.io/configuration/filter-rules/#filter-operators)

**Reply by wc-matteo** [reply]
I know. The main feature would be to negate and/or. The negation of the filter operators was an added bonus (although I think it's much cleaner that way)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
